json/src/test/java/com/baeldung/jsonobject/iterate/JSONObjectIteratorUnitTest.java

```diff
@@ -0,0 +1,79 @@
+package com.baeldung.jsonobject.iterate;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Map;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Test;
+
+public class JSONObjectIteratorUnitTest {
+
+    private JSONObjectIterator jsonObjectIterator = new JSONObjectIterator();
+
+    @Test
+    public void testJsonObjectIterator() {

```

Comment:
Please follow the givenXXX_whenYYY_thenZZZ naming convention described in the style guide.